hi-ml-cpath/src/health_cpath/datasets/default_paths.py

```diff
@@ -4,11 +4,14 @@
 #  ------------------------------------------------------------------------------------------
 
 PANDA_DATASET_ID = "PANDA"
-PANDA_TILES_DATASET_ID = "PANDA_tiles_20220530"
+PANDA_TILES_DATASET_ID_5X = "PANDA_5X_tiles_20220714_134233_level1_224"
+PANDA_TILES_DATASET_ID_20X = "PANDA_20X_tiles_20220714_142937_level0_224"
```

Comment:
[Minor] How about `PANDA_TILES_5/20X_DATASET_ID` or `PANDA_5/20X_TILES_DATASET_ID`? Likewise for the dirs below?